src/ploneintranet/workspace/basecontent/templates/document_view.pt

```diff
@@ -29,8 +29,10 @@
                   data-pat-inject="source:#global-statusmessage; target:#global-statusmessage; hooks: raptor &amp;&amp; source: #application-body; target: #application-body">
               <div class="metadata pat-bumper" id="meta">
                 <div class="meta-bar">
-                  <h1 class="doc-title" id="document-title">${context/title}</h1>
-                  <input name="title" tal:attributes="disabled read_only" type="text" maxlength="70" placeholder="Document title" class="doc-title pat-content-mirror" data-pat-content-mirror="target: #document-title" value="${context/title}" />
+                  <span class="title-group">
+                    <h1 class="doc-title" id="document-title">${context/title}</h1>
+                    <textarea name="title" tal:attributes="disabled read_only" maxlength="70" placeholder="Document title" class="doc-title pat-content-mirror" data-pat-content-mirror="target: #document-title" tal:content="context/title" />
```

Comment:
@pilz  Replacing the input field with a textarea here causes robot test failures, since "I can create a new document" expects an input field here. Since this change probably comes from proto, can I safely update the robot test(s) accordingly?